bin/resources.py

```diff
@@ -27,22 +27,36 @@
                 endpoints[endpoint]["pipelines"][pipeline] = True
                 endpoints[endpoint]["collection"] = row["collection"]
 
+
+old_resources_map = {}
+# Load old-resource.csv to create a mapping of old to updated resources
+for row in csv.DictReader(open("collection/old-resource.csv", newline="")):
+    old_resource = row["old-resource"]
+    updated_resource = row["resource"]
+    old_resources_map[old_resource] = updated_resource
+
 # load resources
 for row in csv.DictReader(open("collection/resource.csv", newline="")):
     resource = row["resource"]
+
+    # Skip this resource if it's an old resource
+    if resource in old_resources_map:
+        continue
     resources[resource] = row
     resources[resource].setdefault("pipelines", {})
     resources[resource]["endpoints"] = row["endpoints"].split(";")
 
     # add collections to the resource
     for endpoint in resources[resource]["endpoints"]:
-        endpoints[endpoint].setdefault("resource", {})
-        endpoints[endpoint][resource] = True
-        resources[resource]["collection"] = endpoints[endpoint]["collection"]
+        if endpoint in endpoints:
```

Comment:
what kind of situation would result in having an endpoint for a resource in resource.csv, but the endpoint not found in endpoint.csv?

Reply:
Just an extra check to make sure only the endpoints which are in the endpoint.csv are processed.